packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Client secret rotation and key management
+meta:
+  - name: description
+    content: Okta client secret rotation helps you rotate and manage your client secrets without service or app downtime. Additionally, you can generate public/private key pairs and manage them using the Admin Console.
+layout: Guides
+sections:
+- main
+---
+
+<ApiLifecycle access="ea" />
+
+This guide shows you how to rotate and manage your client secrets without service or application downtime. Additionally, this guide shows you how to generate public/private key pairs and manage them using the Admin Console.
+
+---
+
+**Learning outcomes**
+
+* Create additional client secrets for a client app and manage existing client secrets
+* Generate public/private key pairs for a client app using the Admin console and manage additional key pairs
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* **OAuth secrets and key management** feature enabled in the Admin Console (**Settings** | **Features**).
+* Access to the new APIs: Client Secret Management `/api/v1/apps/{appId}//credentials/secrets` and JWKS Management `/api/v1/apps/{appId}/credentials/jwks`
+* An existing OpenID Connect client app in Okta for testing and an existing Service client app in Okta (optional)
+[Postman client](https://www.getpostman.com/downloads/) to test requests. See [Get Started with the Okta APIs](https://developer.okta.com/code/rest/) for information on setting up Postman.
+* The [Client Secret Rotation and JWT Creation] Postman collection that allows you to test the API calls that are described in this guide. Click **Run in Postman** to add the collection to Postman.
+
+  [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/8e9d91cef0d5ab9e9fa7)
+
+---
+
+## About client secret rotation and public/private key pairs
+
+Just like periodically changing passwords, regularly rotating the client secret that your application uses to authenticate is a security best practice. The challenge with rotating the client secret is to facilitate a seamless client secret rotation without service or application downtime. You need the ability to create overlapping client secrets.
+
+Additionally, depending on what type of credentials that a [client uses to authenticate](/docs/reference/api/oidc/#client-authentication-methods), the use of a JWK public/private key pair may be required. Applications that use private/public key pairs for client authentication have substantially higher security because the private key can only be accessed by the client. But, private/public key pair generation can be laborious and time-consuming, and using the API can lead to errors.
```

Comment:
Question: I notice that you use "application" up to this point, then use "app" from here on. How come? I'm still trying to figure out how and when to use "app".